checkout-ui-custom/src/_js/_v.custom.checkout.ui.js

```diff
@@ -1082,7 +1082,10 @@ class checkoutCustom {
 
     $('body').on('click', '.show-more-items-button', function () {
       _this.general()
+      _this.update(_this.orderForm)
+      vtexjs.checkout.getOrderForm()
```

Comment:
Why are you calling `getOrderForm` after updating? 

Could you fix the lint errors by changing to `window.vtexjs.checkout.getOrderForm()` and deleting the extra line in 1088?

Reply:
Anna, I call it to update the UI and then rebuild as expected. I know there's some room to improvements just calling the functions 

    vcustom.checkout.setParentIndex(vtexjs.checkout.orderForm)
    vcustom.checkout.indexedInItems(vtexjs.checkout.orderForm)

Howver, few things are missing from the previously script. So it's better to refresh the UI